gno.land/cmd/gnoland/testdata/event_defer_callback_loop.txtar

```diff
@@ -0,0 +1,68 @@
+# start a new node
+gnoland start
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/demo/edcl -gas-fee 1000000ugnot -gas-wanted 2500000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2500000'
+stdout 'GAS USED: [0-9]+'
+stdout 'HEIGHT: [0-9]+'
+stdout 'EVENTS: \[\]'
```

Comment:
@gfanton can you explain how to use your new feature?

Reply:
There should be other tests already using `loadpkg`.

Reply:
Updated 

https://github.com/gnolang/gno/pull/1653/commits/7a5fd741d4b2fbd5857b8055abb6c08c8c11e4be